gsa/src/web/components/menu/menu.js

```diff
@@ -114,13 +116,24 @@ const MenuList = styled.ul`
   }
 `;
 
+const checkChild = child => {
+  if (child.type === MenuSection && isArray(child.props.children)) {
+    return child.props.children.find(chil => chil !== false);
```

Comment:
I am really sure this won't work always. Please use the React.Children API (https://reactjs.org/docs/react-api.html#reactchildren) and I guess null and undefined should be checked too. Maybe use
```javascript
chil => !!chil
```